<?xml version="1.0"?>
<launch>
	<node name="range_image" type="aiimooc_syz_node" pkg="aiimooc_syz" output="screen" respawn="false">
	</node>
    <node name="rviz" pkg="rviz" type="rviz"  required="true" />  
    
</launch>
